NagyAdrian/teme/3.3-obiecte/script.js

```diff
@@ -0,0 +1,59 @@
+// Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată
+
+// Input: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]
+
+// Output: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }
+
+let reteta = { titlu: "", portii: "", ingredinte: [] }
+function Scriere(o) {
+  o.titlu = "pizza"
+  o.portii = "3"
+  o.ingrediente = ["sunca", "cas", "ketcup"]
+}
+
+Scriere(reteta)
```

Comment:
Deci e posibil ca enuntul sa nu fi fost suficient de clar. 
Parametrii de intrare al functiei, ar trebuii sa fie, titlu, portii si ingrediente.
![image](https://user-images.githubusercontent.com/45266485/201028183-fd3c4827-cc77-4caf-958e-c1b2e26ec8b5.png)

Reply:
Numele functiei trebuie sa fie descriptiv.  "Scriere" nu e o denumire buna. Un lucru de observat, e ca toate denumirile de variabile si functii, ar trebuii sa fie in engleza. Codul de la un job, e extrem de improbabil sa fie in romana, si daca colegii tai scriu in engleza codul si tu in romana... nu va iesii bine. E o chestie de obisnuinta, si best practice.

Reply:
![image](https://user-images.githubusercontent.com/45266485/201029248-d61b09e4-7919-4821-bb61-d496a611bcd4.png)
Trebuie sa faci un console log pentru fiecare proprietate a obiectului.